tests/test_atom_mapping.py

```diff
@@ -1,67 +1,426 @@
+   -0.6439   -1.3144   -0.1494 C   0  0  0  0  0  0  0  0  0  0  0  0
+    0.4262   -1.2251    0.6719 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.2949    1.4641    0.2937 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.6968    1.9715   -1.2775 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.0662   -0.1837   -1.3042 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.4970   -0.3613   -2.0575 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.6428   -1.9811    1.4121 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.2637   -2.1987   -0.1345 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.4850    1.5611   -0.6965 H   0  0  0  0  0  0  0  0  0  0  0  0
+    0.8877    1.9212    0.8230 H   0  0  0  0  0  0  0  0  0  0  0  0
+    1.8010    0.1189    1.4889 H   0  0  0  0  0  0  0  0  0  0  0  0
+    2.1753   -0.3430   -0.0537 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.2711   -0.8618    0.6186 H   0  0  0  0  0  0  0  0  0  0  0  0
+   -0.1781   -0.6848    1.4006 H   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0
+  1  3  1  0
+  1 13  1  0
+  1 14  1  0
+  2  6  1  0
+  2 15  1  0
+  2 16  1  0
+  3  4  1  0
+  3  7  1  0
+  3  8  1  0
+  4  5  1  0
+  4  9  1  0
+  4 10  1  0
+  5  6  1  0
+  5 12  1  0
+  5 17  1  0
+  6 11  1  0
+  6 18  1  0
+M  END
+$$$$""",
+        removeHs=False,
+    )
+    return mol_a, mol_b
+
+
+def test_cyclohexane_stereo():
+    # test that cyclohexane in two different conformations has a core alignment that is stereo correct. Note that this needs a
+    # larger than typical cutoff.
+    mol_a, mol_b = get_cyclohexanes_different_confs()
+    all_cores = atom_mapping.get_cores(
+        mol_a,
+        mol_b,
+        ring_cutoff=0.15,
+        chain_cutoff=0.30,
+        max_visits=1e6,
+        connected_core=True,
+        max_cores=100000,
+        enforce_core_core=True,
+        complete_rings=True,
+    )
+    print(len(all_cores))
+
+    for core_idx, core in enumerate(all_cores[:1]):
+        res = plot_atom_mapping_grid(mol_a, mol_b, core, num_rotations=5)
+        with open(f"atom_mapping_0_to_1_core_{core_idx}.svg", "w") as fh:
+            fh.write(res)
+
+    # 1-indexed
+    expected_core = np.array(
+        [
+            [1, 1],  # C
+            [2, 2],  # C
+            [3, 3],  # C
+            [4, 4],  # C
+            [5, 5],  # C
+            [6, 6],  # C
+            [16, 14],  # C1H
+            [15, 13],  # C1H
+            [17, 15],  # C2H
+            [18, 16],  # C2H
+            [7, 7],  # C3H
+            [8, 8],  # C3H
+            [9, 9],  # C4H
+            [10, 10],  # C4H
+            [14, 17],  # C5H
+            [11, 12],  # C5H
+            [13, 18],  # C6H
+            [12, 11],  # C6H
+        ]
+    )
+
+    # 0-indexed
+    expected_core -= 1
+
+    all_cores_fzset = get_all_cores_fzset(all_cores)
+    assert tuples_to_set(expected_core) in all_cores_fzset
+
+    # TBD: if we have check_chiral == True then we should expect length of all_cores == 1
```

Comment:
@maxentile added a placeholder for you to ensure that if we check for chirality then only a single core-mapping should be valid.

Reply:
Thanks -- will take a closer look after the holiday!

(As of yesterday morning, was intending to re-enable the chiral check disabled by this PR in a subsequent PR using this branch: https://github.com/proteneer/timemachine/compare/bgl...chiral_check . Will also update any affected tests.)